src/TextThought.py

```diff
@@ -605,8 +606,8 @@ def _create_textview(self):
         """
 
         r, g, b = utils.gtk_to_cairo_color(self.foreground_color)
-        rgba = Gdk.Color(int(65535 * r), int(65535 * g), int(65535 * b))
-        self.textview.modify_text(Gtk.StateType.NORMAL, rgba)
+        rgba = Gdk.RGBA(int(65535 * r), int(65535 * g), int(65535 * b))
+        self.textview.modify_text(Gtk.StateType.NORMAL, rgba.to_color())
```

Comment:
I'm expecting to get an error here and I'll look into it

Reply:
Your code is in error, but I would not expect an error message.  It is another one of those situations where code can execute but the result is not what you planned.

The values are _clamped_ to the range 0.0 to 1.0, as can be seen in the [source code for Gdk.RGBA](https://github.com/GNOME/gtk/blob/bb56b4ef5d975fff91e7caa39e058d02fd7373bb/gdk/gdkrgba.c#L376).  Always look at the source code of GTK if you have any doubts as to what it does; the PyGObject documentation is often incomplete.

Use interactive testing with a Python REPL to prove what you expect;

```log
>>> from gi.repository import Gdk
>>> Gdk.RGBA(65535, 65535, 65535).to_string()
'rgb(255,255,255)'        <-- correct answer for the wrong reason due to clamping
>>> Gdk.RGBA(65535, 65535, 1).to_string()
'rgb(255,255,255)'        <-- correct answer but you may not have expected it
>>> Gdk.RGBA(65535, 65535, 0.5).to_string()
'rgb(255,255,128)'        <-- correct answer, see how BLUE is at 50% intensity
>>> Gdk.RGBA(65535, 65535, 0.5).blue
0.5        <-- can read the BLUE channel
>>> Gdk.RGBA(65535, 65535, 65535).blue
65535.0      <-- shows the clamping is happening _after_ the object is created
>>> 
```